src/mexpr.jl

```diff
@@ -90,22 +129,22 @@ const m_to_jl = Dict("%e" => "e",
     "inf"   =>  "Inf",
     "minf"  =>  "-Inf")
 
+const m_to_jl_utf = Dict(":" => "=")
+
 const jl_to_m = Dict("e" => "%e",
     "eu" => "%e",
     "pi" => "%pi",
-    "π" => "%pi",
+    "π" => "pi",
```

Comment:
this change was necessary in order to avoid a double percentage sign to be produced